### ISSUE TITLE
False positive not-callable when __call__ method is inherited

### ISSUE THREAD

**Issue by joshburkart** [PyCQA/pylint#897]
The last line produces a "not-callable" lint error:

```python
class CallableSubclass(another_module.CallableClass):
    pass

CallableSubclass()(7)
```

**Comment by PCManticore** [comment]
Hi. I can't reproduce this error, using a similar sample of code. Can you provide more information about another_module, is it a C extension? Can pylint access it, e.g. you're not getting an import-error for it?

**Comment by joshburkart** [comment]
Ahh my bad. I'm using a fork of pylint specific to my company. Probably a problem with the fork. Sorry for the trouble!

**Comment by PCManticore** [comment]
Oh, okay, no worries!
